CHANGELOG.md

```diff
@@ -8,7 +8,12 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 -->
 
-## Unreleased
+The latest published Bevy Vello release is [0.5.1](#051---2024-07-04) which was released on 2024-07-16.
```

Comment:
This date in the text was copied from Vello, and I believe it is not correct for this library.

```suggestion
The latest published Bevy Vello release is [0.5.1](#051---2024-07-04) which was released on 2024-07-04.
```